src/App.tsx

```diff
@@ -1,61 +1,78 @@
 import './App.scss';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { Todo } from './Types/Todo';
+import todosFromServer from './api/todos';
+import { getUserById } from './services/user';
+import { getNewTodoId } from './services/id';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const initialTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
```

Comment:
Ensure that `initialTodos` is correctly initialized with user data. The `getUserById` function is used here, which is appropriate as per the checklist to prepare data in one place (App component) and pass it to child components.